package.json

```
@@ -11,5 +11,11 @@
   "bugs": {
     "url": "https://github.com/ws-4020/rn-spoiler/issues"
   },
-  "homepage": "https://github.com/ws-4020/rn-spoiler#readme"
+  "homepage": "https://github.com/ws-4020/rn-spoiler#readme",
+  "scripts": {
+    "create-hands-on-app": "node example/hands-on/create-hands-on-app.js"
```

あとでいくつかできる可能性を考えて、`create-app:*`って感じで指定できるようにしておくと、稀に便利なことがあります。

```suggestion
    "create-app:hands-on": "node example/hands-on/create-hands-on-app.js"
```